android/src/main/java/com/reactnativestripesdk/CollectBankAccountLauncherFragment.kt

```diff
@@ -38,10 +38,10 @@ class CollectBankAccountLauncherFragment(
     super.onCreate(savedInstanceState)
 
     val stripeSdkModule: StripeSdkModule? = context.getNativeModule(StripeSdkModule::class.java)
-    if (stripeSdkModule != null && stripeSdkModule.eventListenerCount > 0) {
```

Comment:
I dropped tracking the listener count, as there is no api for it with the new codegened event emitters and doesn't seem to be needed.